ops-catalog/template-local.flow.yaml

```diff
@@ -2,6 +2,22 @@ import:
   - template-common.flow.yaml
 
 materializations:
+  estuary/global/inferred-schemas:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/materialize-postgres:dev
+        config: stats-local-endpoint.sops.yaml
+
+    bindings:
+      - source: estuary/global/schema-inference-L2
+        resource:
+          table: catalog_schemas
```

Comment:
I'll suggest `inferred_schemas` for the table name, since they may or may not actually be used in the catalog. That would also avoid any potential confusion with [the Catalog schema](https://github.com/estuary/flow/blob/master/flow.schema.json), which is the schema of a Flow catalog spec.

Reply:
Yep. Thought I heard Johnny wanted it to be called `catalog_schemas` but this is better 👍